sfepy/solvers/nls.py

```diff
@@ -264,6 +264,9 @@ class Newton(NonlinearSolver):
             solve."""),
         ('is_linear', 'bool', False, False,
          'If True, the problem is considered to be linear.'),
+        ('scaled_error', 'bool', False, False,
+         """If True, the error of the linear solver is calculated using
+            the scaled values."""),
```

Comment:
Maybe put this below the lin_precision option as it plays with that? Or below scale_solution_fun (where the line is too long, mea culpa :))